en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store uses an application that captures online orders. For each order, the application also needs to generate a notification for the sales manager as well as update the inventory recordsinventory records and notify the status to the client who places the order.
+
+## Before you begin
+
+The following are required to try out this tutorial:
+
+- A Google account. In this Google account, open the Google Drive and create a Google sheet named `inventory` with the following table:
+
+    ![Google Sheet Extract](../assets/img/tutorials/inventory-service-google-sheet-extract.png){.cInlineImage-half}
```

Comment:
This image is not rendering in the GitHub view file option. May be some additional spaces at the beginning of the line. Please check and correct.